index.bs

```diff
@@ -10366,13 +10368,11 @@ is the [=interface=]:
         |I|.
 </ol>
 
-If the internal \[[Call]] method
-of the [=interface object=]
-returns normally, then it must
-return an object that implements interface |I|.
+If evaluating the [=function object=] |F| returns normally,
+then it must return an object that implements interface |I|.
```

Comment:
Hmm. This requirement is a little strange, both before and after. I think what it is trying to say is instead "If the actions listed in the description of _constructor_ return normally, then they must return an object that implements interface _I_."